base/files/file.cc

```diff
@@ -31,12 +31,24 @@ File::Info::~Info() = default;
 File::File()
     : error_details_(FILE_ERROR_FAILED),
       created_(false),
-      async_(false) {
+      async_(false)
+#if defined(STARBOARD)
+      ,
+      append_(false)
```

Comment:
Our footprint would be smaller if we defaulted `append_ = false` in the header and only set it in the copy constructors